exercises/grade-school/Example.cs

```diff
@@ -13,7 +13,21 @@ public void Add(string student, int grade)
             roster.Add(grade, new SortedList<string> { student });
     }
 
-    public IEnumerable<string> Roster(int grade) => roster[grade];
+   public IEnumerable<string> Roster()
+   {
+    var students = new List<string>(); 
```

Comment:
The indentation of all lines in the `Roster` method are off. They should all be indented one tab more.